tcms/telemetry/api.py

```diff
@@ -155,10 +164,17 @@ def execution_trends(query=None):
     for _key, value in data_set.items():
         del value[0]
 
+    all_count = status_count['pass'] + status_count['fail']
```

Comment:
This calculation is wrong. The number of all statuses includes neutral statuses too. Best is to probably call `.count()` on the query above.